onebusaway-android/src/main/res/layout/fragment_trip_plan.xml

```diff
@@ -0,0 +1,323 @@
+                                    android:layout_column="2"
+                                    android:textColor="@color/body_text_2"
+                                    android:background="@drawable/bottom_line_grey"
+                                    android:textSize="20dp"
+                                    android:gravity="center_horizontal"
+                                    android:linksClickable="true"
+                                    android:longClickable="false"
+                                    android:layout_weight="1"
+                                    android:textIsSelectable="false"
+                                    android:layout_marginLeft="2dp"
+                                    android:layout_marginRight="2dp"/>
+
+                            <TextView
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="30dp"
+                                    android:textAppearance="?android:attr/textAppearanceSmall"
+                                    android:background="@drawable/bottom_line_grey"
+                                    android:id="@+id/date"
+                                    android:textColor="@color/body_text_2"
+                                    android:textSize="20sp"
+                                    android:textIsSelectable="false"
+                                    android:layout_weight="1"
+                                    android:gravity="center_horizontal"
+                                    android:linksClickable="true"
+                                    android:layout_marginLeft="2dp"
+                                    android:layout_marginRight="2dp"/>
+
+                            <TextView
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="30dp"
+                                    android:textAppearance="@android:style/TextAppearance.Medium"
+                                    android:id="@+id/time"
+                                    android:layout_column="2"
+                                    android:background="@drawable/bottom_line_grey"
+                                    android:textColor="@color/body_text_2"
+                                    android:textSize="20dp"
+                                    android:singleLine="false"
+                                    android:layout_weight="1"
+                                    android:gravity="center_horizontal"
+                                    android:linksClickable="true"
+                                    android:layout_marginLeft="2dp"
+                                    android:layout_marginRight="2dp"/>
+
+                        </LinearLayout>
+
+                    </LinearLayout>
+                </FrameLayout>
+
+                <ImageButton
+                        android:background="@color/theme_primary"
```

Comment:
Please remove all the styling on this button, including the background, padding, and src image (and remove the `@drawable/plan_my_trip` from the project) - it should just be a normal material design button with the text "Plan my trip".